apps/dashboard/src/pages/api/testnet-faucet/eligibility.ts

```diff
@@ -0,0 +1,27 @@
+import type { NextApiRequest, NextApiResponse } from "next";
```

Comment:
move his to api route handlers instead of adding to pages/api (deprecated)

see: https://nextjs.org/docs/app/building-your-application/routing/route-handlers